Privacy Policy - jsdelivr.com.md

```diff
@@ -190,42 +190,6 @@ to provide the Service on our behalf, to perform Service-related services or to
 These third parties have access to your Personal Data only to perform these tasks on our behalf
 and are obligated not to disclose or use it for any other purpose.
 
-### Analytics
-
-We may use third-party Service Providers to monitor and analyze the use of our Service.
-
-#### Google Analytics
-
-Google Analytics is a web analytics service offered by Google that tracks and reports website traffic.
```

Comment:
This may need to be replaced by Plausible, or do we self host that?

Reply:
Self host